### ISSUE TITLE
Maximizing clickable area for checkboxes and radio buttons across platform

### ISSUE THREAD

**Issue by substrae** [GCTC-NTGC/gc-digital-talent#8965]
## 🕵️‍♀️ Details (updated for dev)

Bigger clickable ares are easier to click, it's as simple as that - and it can make a big different for the ease of use for some users.

This issue aims to make the clickable area as big as possible for checkboxes and radio buttons with the existing component size. 

From Josh's proposed solution below:
> in order to improve the target area for checkboxes and radio circle specifically, we can apply a custom design that converts the existing whitespace to clickable area, expanding its target size. If we take the custom approach, we can also increase the checkbox/radio option height and width to x.75, effectively increasing the total target area to 34px tall and 40.5px wide.

**Updated clickable area:**
![Image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/2494551/4b2bf812-f3ba-4142-ac07-d288c38bee5a)

**True clickable area (including label):**
![Image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/2494551/b34cfc39-79c9-4767-87a0-ea880825660f)

**Designer**
@substrae 

💻 Related WCAG Criteria
•	2.5.5 Target Size: The size of the [target](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html#dfn-target) for [pointer inputs](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html#dfn-pointer-input) is at least 44 by 44 [CSS pixels](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html#dfn-css-pixel).

## Acceptance criteria

- [ ] Line spacing converted to clickable area (see comments below)
- [ ] Padding around checkbox group converted to clickable area (see comments below)
- [ ] Visual appearance of radio button circle and checkbox box (height and width) increased to x.75
- [ ] Review from @substrae

**Comment by tristan-orourke** [comment]
Will this need to change visible whitespace?
What about multiline items?

**Comment by gobyrne** [comment]
@substrae could you flush this out a little? The devs thought that all the area around the checkboxes was already clickable and that increasing the clickable area would mean increasing the space between lines. Also, multiline items ilke Tristan is aking.

**Comment by substrae** [comment]
@gobyrne @tristan-orourke 

Not sure I follow the questions... The area around checkboxes _should_ already by clickable, yes, but this was about increasing the visual click target, as far as I understood. The custom box/circle eats into the existing whitespace, because the existing whitespace is as high as it can be without incurring too much space between items. I've added a visual representation of what the clickable area should be, represented by the blue boxes in the right-hand image. The click target for single line items ends up being `40.5px`, which gets us as close to AAA as we can get.

Multiline items should wrap as pictured.

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/6960386/b59bd30a-d2fd-4cc0-9f6e-c700f73cfa49)
